ms2rescore/id_file_parser.py

```diff
@@ -67,6 +66,7 @@ def __init__(self, config: Dict, output_basename: Union[str, os.PathLike]) -> No
             "generic": r".+_([0-9]+)_[0-9]+_[0-9]+",
             "tandem": r".+_([0-9]+)_[0-9]+_[0-9]+",
             "msgfplus": r".+_SII_([0-9]+)_[0-9]+_[0-9]+_[0-9]+",
+            "USI": r"mzspec:PXD[0-9]{6}:[^\s\:]*:scan:([0-9]+)"
```

Comment:
we should enable the use of the universal spectrum identifier. Either we can use this as a pattern but the problem is that when we do this there can be non-unique indexes that cause an error in MS²Rescore. Another option is to set extract_spectrum_index to false, this way they stay unique